gozo-stores/src/components/FiltersSidebar.jsx

```diff
@@ -1,50 +1,69 @@
                     </div>
                     <div class="price-amount">
-                        <p>Price Amount</p>
+                        <p>{state.priceRange}</p>
                     </div>
                 </div>
 
                 <div class="filter" id="category">
                     <div class="category-checkbox flex-col">
                         <label>Category</label><br />
-                        <span><input type="checkbox" name="category" id="batman" />Marvel</span>
-                        <span><input type="checkbox" name="category" id="ironman" />DC</span>
-                        <span><input type="checkbox" name="category" id="joker" />DCEU </span>
-                        <span><input type="checkbox" name="category" id="joker" />Others </span>
+
+                        <span><input type="checkbox" name="category" value="marvel" onChange={(e) => dispatch({type: "ADD-CAT", payload: e.target.value, status: e.target.checked})}  />Marvel</span>
+
+                        <span><input type="checkbox" name="category" value="dc"
+                        onChange={(e) => dispatch({type: "ADD-CAT", payload: e.target.value, status: e.target.checked})}  />DC</span>
+
+                        <span><input type="checkbox" name="category" value="dceu"
+                        onChange={(e) => dispatch({type: "ADD-CAT", payload: e.target.value, status: e.target.checked})}  />DCEU </span>
+                        
+                        <span><input type="checkbox" name="category" value="others"
+                        onChange={(e) => dispatch({type: "ADD-CAT", payload: e.target.value, status: e.target.checked})}  />Others </span>
+
                     </div>
                 </div>
 
                 <div class="filter" id="rating">
                     <div class="rating-radio-button flex-col">
                         <label>Rating</label>
-                        <span><input type="radio" name="rating" id="4+" />4+</span>
-                        <span><input type="radio" name="rating" id="3+" />3+</span>
-                        <span><input type="radio" name="rating" id="2+" />2+</span>
-                        <span><input type="radio" name="rating" id="2+" />1+</span>
+                        <span><input type="radio" name="rating" value="4" 
+                        onChange={(e) => dispatch({type: "RATING", payload: e.target.value})}/>4+</span>
+                        <span><input type="radio" name="rating" value="3" 
+                        onChange={(e) => dispatch({type: "RATING", payload: e.target.value})}/>3+</span>
+                        <span><input type="radio" name="rating" value="2" 
+                        onChange={(e) => dispatch({type: "RATING", payload: e.target.value})}/>2+</span>
+                        <span><input type="radio" name="rating" value="1"
+                        onChange={(e) => dispatch({type: "RATING", payload: e.target.value})}/>1+</span>
```

Comment:
Use the map function for rating. These will reduce your lines of code too.

Reply:
yeah that's a good idea, as everything is same for them